spec/system/folder_action_system_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'FolderAction', type: :system, js: true do
+  include SystemHelpers
+  # can select folder and team from side-navbar
+  it 'opens folder and team in side-navbar' do
+    login_as_user(:admin)
+    visit('/')
+
+    within(page.find('#side-bar-team-235930340')) do
```

Comment:
So, [Magic Numbers](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) are never really a good thing. 
In this case I read through the test and do not really have a clue what is going on with this crazy number.
Now, granted there are other examples of this in our test suite so you shouldn't feel too bad for doing it and it's certainly not the end of the world. 
Another thought I had was that the fixture might change that very specific ID at some point in the future but I found out that ActiveRecord Fixtures IDs are apparently [static](https://api.rubyonrails.org/v3.1/classes/ActiveRecord/Fixtures.html#:~:text=The%20generated%20ID%20for%20a,as%20we%20know%20the%20label.) so by this regard this is probably fine. 
Nevertheless, this can easily be evaded by some string interpolation making the test more readable as well in the process -> ex.: `within(page.find("#side-bar-team-#{Team.first.id}))` this works but could be made even more readable by getting the team straight from the logged-in user. Since, from what I can gather the user only has one team so just calling `Team.first` works in this case but also isn't really specific and what if the first team changes in the future? (this is left as an experiment to the reader :stuck_out_tongue_winking_eye:  ).